Dockerfile-Dev

```diff
@@ -20,24 +18,41 @@ FROM mcr.microsoft.com/dotnet/core/sdk:3.1 as helium-dev-base
 # ENV no_proxy {value}
 # ENV NO_PROXY {value}
 
-### install azure cli
+ENV DEBIAN_FRONTEND=noninteractive
+
+### install dependencies
 RUN apt-get update && \
-    apt-get install -y apt-transport-https ca-certificates curl software-properties-common libssl-dev git wget nano lsb-release jq && \
-    curl -sL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/microsoft.asc.gpg && \
+    apt-get install -y sudo apt-utils apt-transport-https ca-certificates curl gnupg software-properties-common libssl-dev git wget nano lsb-release jq && \
+    apt-get upgrade -y
+
+### install azure cli
+RUN curl -sL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/microsoft.asc.gpg && \
     CLI_REPO=$(lsb_release -cs) && \
     echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ ${CLI_REPO} main" > /etc/apt/sources.list.d/azure-cli.list && \
     apt-get update && \
     apt-get install -y azure-cli
 
-RUN mkdir -p /root/.azure
+### add helium user
+RUN useradd -u 1001 -U -m -s /bin/bash helium && \
+    usermod -aG sudo helium && \
+    gpasswd -a helium sudo &&  \
```

Comment:
do we need to update readme to call out potentially needing to change this number? (can be done with a later readme task)

Reply:
updated readme